# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/typeaheadfind/nsTypeAheadFind.cpp
# Commit: 8bf69c3fd814
# Full Hash: 8bf69c3fd81419337cb93f96020bb5e88a333d32
# Author: Jan Varga <Jan.Varga@gmail.com>
# Date: 2012-08-07 13:39:24
# Regressor Bug: 779684
# File Overlap Count: 1
# Description:
#   Bug 779684 - Move FrameIteratorFlags to nsFrameIterator class. r=roc
# ==============================================================================

diff -r 78aaaf0d8185 -r 8bf69c3fd814 toolkit/components/typeaheadfind/nsTypeAheadFind.cpp
--- a/toolkit/components/typeaheadfind/nsTypeAheadFind.cpp	Mon Aug 06 14:51:33 2012 -0600
+++ b/toolkit/components/typeaheadfind/nsTypeAheadFind.cpp	Mon Aug 06 20:27:04 2012 -0400
@@ -1115,7 +1115,7 @@
   // view port. Move range forward to first visible point,
   // this speeds us up a lot in long documents
   nsFrameIterator frameTraversal(aPresContext, frame,
-                                 eLeaf, FrameIteratorFlags::FLAG_NONE);
+                                 eLeaf, nsFrameIterator::FLAG_NONE);
 
   while (rectVisibility == nsRectVisibility_kAboveViewport) {
     frameTraversal.Next();
